pkg/naming/main.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 // ConfigMap builds the name for the config map used in the AmazonCloudWatchAgent containers.
-func ConfigMap(agent v1alpha1.AmazonCloudWatchAgent) string {
-	return DNSName(Truncate("%s", 63, agent.Name))
+func ConfigMap(_ v1alpha1.AmazonCloudWatchAgent) string {
+	return "cwaagentconfig"
```

Comment:
I guess this is okay, but this is also assuming that everyone will use the ACWA resource defined in helm. If we ever change that cm name then everything will break.

Reply:
Yeah this is just for the config map that the operator creates and mounts. Users currently shouldnt be even aware of this currently and cannot override it without changing the operator name itself. So we are defining this behavior for the first time and I guess we are saying we will always call this what our current public docs do.